src/types.ts

```diff
@@ -1,13 +1,38 @@
-export type Post = {
+export type ApiPost = {
```

Comment:
This is the raw type representing what cosmic sends us back, you see all of the spanish and soundcloud stuff is nested inside of `metadata`, but that all gets mapped into the more convenient `DualPost` type in the api file.